src/bin.js

```diff
@@ -30,6 +30,12 @@ const OPTIONS = {
     default: ARCHIVE_FORMAT_ZIP,
     describe: 'Format of the archive created for each function',
   },
+  config: {
+    object: true,
```

Comment:
I think `object` is not a known property with Yargs (although it might ignore unknown properties).
Yargs automatically parses dot notation CLI flags to objects (unless the `dot-notation` parser property is used).

Reply:
Thanks! Removed in https://github.com/netlify/zip-it-and-ship-it/pull/426/commits/966e8f25fac8248f6e4d56a9f0d226cdced78672.